packages/discord.js/package.json

```diff
@@ -52,7 +52,7 @@
     "@discordjs/rest": "workspace:^",
     "@sapphire/snowflake": "^3.1.0",
     "@types/ws": "^8.2.2",
-    "discord-api-types": "^0.27.0",
+    "discord-api-types": "^0.27.1",
```

Comment:
/builders bumps this to 0.27.2, but this only bumps to 0.27.1.

Should be bumped to 0.27.3, regardless.